Depfile

```diff
@@ -0,0 +1,8 @@
+---
+go:
+  buf:
+    importPath: "github.com/bufbuild/buf/cmd/buf"
+    version: "v1.9.0"
```

Comment:
We should use the latest versions as defined in: https://github.com/aserto-dev/deps/blob/main/deps/Depfile

Reply:
Used go 1.19